cli/js/net.ts

```diff
@@ -123,6 +148,50 @@ export class ListenerImpl implements Listener {
   }
 }
 
+export class UDPConnImpl implements UDPConn {
+  constructor(
+    readonly rid: number,
+    readonly addr: Addr,
+    public bufSize: number = 1024,
+    private closing: boolean = false
+  ) {}
+
+  async receive(): Promise<[Uint8Array, Addr]> {
+    const p = new Uint8Array(this.bufSize);
+    const { size, remoteAddr } = await sendAsync(
+      dispatch.OP_RECEIVE,
+      { rid: this.rid },
+      p
+    );
```

Comment:
I think this op should match [libc naming and semantics](https://linux.die.net/man/2/recvfrom).
```c
     ssize_t
     recvfrom(int socket, void *restrict buffer, size_t length, int flags,
         struct sockaddr *restrict address, socklen_t *restrict address_len);
```

We ought to also expose it directly: 
```
let bytesReceived = await Deno.recvfrom(rid, buf, addr)
```